tests/roles/linux-system-roles.rhc

```diff
@@ -0,0 +1 @@
+../..
```

Comment:
making `linux-system-roles.rhc` the symlink will cause circular references - instead, create the directory `tests/roles/linux-system-roles.rhc` and in this directory create symlinks to `../../../tasks`, `../../../defaults`, etc.